service/src/main/java/com/appirio/service/billingaccount/api/PublisherSQS.java

```diff
@@ -0,0 +1,38 @@
+package com.appirio.service.billingaccount.api;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicSessionCredentials;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+public class PublisherSQS implements PublisherConsumedAmount
+{
+    private static final Logger logger = LoggerFactory.getLogger(PublisherSQS.class);
+    public void publish(String msg)
+    {
+        try
+        {
+            String awsAccessKey = System.getenv("AWS_ACCESS_KEY_ID");
+            String awsSecretKey = System.getenv("AWS_SECRET_ACCESS_KEY");
+            String awsSessionToken = System.getenv("AWS_SESSION_TOKEN");
+            String urlSQS = System.getenv("AWS_SQS_URL");
+            AmazonSQS sqsClient = AmazonSQSAsyncClientBuilder.standard()
+                .withCredentials(
+                    new AWSStaticCredentialsProvider(
+                        new BasicSessionCredentials(awsAccessKey, awsSecretKey, awsSessionToken)))
+                .withRegion(Regions.US_EAST_1).build();
```

Comment:
I think you shouldn't set the AWSStaticCredentialsProvider. By default the client will lookup the credentials by all means (like env, config file, metadata endpoint etc).